sqlair.go

```diff
@@ -26,48 +24,18 @@ type M map[string]any
 var ErrNoRows = sql.ErrNoRows
 var ErrTXDone = sql.ErrTxDone
 
-var stmtIDCount int64
-var dbIDCount int64
```

Comment:
The benefit of keeping this is that it is much easier to read the map definitions. I would advocate for keeping it if @manadart agrees.
```go
map[uint64]map[uint64]*sql.Stmt
```
vs
```go
map[stmtID]map[dbID]*sql.Stmt
```

Reply:
Haha, he suggested I remove these. They do make the type signature clearer but I don't think its too big a deal. The types always appear next to a variable name that makes this clear.